actionpack/lib/action_dispatch/http/cache.rb

```diff
@@ -8,6 +8,9 @@ module Cache
       module Request
         HTTP_IF_MODIFIED_SINCE = "HTTP_IF_MODIFIED_SINCE"
         HTTP_IF_NONE_MATCH     = "HTTP_IF_NONE_MATCH"
+        HTTP_CACHE_CONTROL    = "HTTP_CACHE_CONTROL"
```

Comment:
nitpick: Using constants like this is a thing that predates frozen string literals, it isn't really preferable anymore today.